pkg/controllers/syncer/translator/namespaced_translator.go

```diff
@@ -17,6 +19,10 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+var (
+	LabelPrefix = "vcluster.loft.sh/label"
```

Comment:
Please get rid of this as this couldn't be used correctly with multi-namespace syncer as we don't want to replace the labels there